### PR TITLE
feat: add `stats/base/dists/planck/mean`

### PR DESCRIPTION
**Pull request by Jaysukh-409**
## Description

> What is the purpose of this pull request?

This pull request:

- Adds the `mean` for the Planck Discrete Random Variable.

## Related Issues

> Does this pull request have any related issues?

This pull request:

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+# Mean
+
+> Planck distribution [expected value][expected-value].
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+The [expected value][expected-value] for a geometric random variable is
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md]: I believe that this sentence needs to be updated.

Comment (kgryte) [reply lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md]: This package isn't for a geometric RV.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+## Usage
+
+```javascript
+var mean = require( '@stdlib/stats/base/dists/planck/mean' );
+```
+
+#### mean( lambda )
+
+Returns the [expected value][expected-value] of a planck distribution with shape parameter `lambda`.
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md]: Same comment regarding s/planck/Planck/.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/base/randu' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md]: Same comment regarding rv generation.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var lambda;
+	var y;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		lambda = ( randu()*20.0 ) + EPS;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js]: Same comment regarding rv generation.

**Review by kgryte — CHANGES_REQUESTED** [review]
Most of the same requested changes in https://github.com/stdlib-js/stdlib/pull/3896 are applicable here.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js (hunk 4 above)
Comment (Jaysukh-409) [reply lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js]: I think to generate `lambda` value we can't use `discrete-uniform` because it is continuous variable.

**Comment by Jaysukh-409** [comment]
If generating rv doesn't need changes then this PR is ready to merge.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js (hunk 4 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js]: @Jaysukh-409 Then you should use `random/array/uniform`. We have various PRNGs, not just discrete uniform. See `random/array/*`. In this case, you just need `uniform( EPS, 20.0 )`.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md (hunk 3 above)
Comment (kgryte) [reply lib/node_modules/@stdlib/stats/base/dists/planck/mean/README.md]: @Jaysukh-409 Use `random/array/uniform`.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/random/base/randu' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/benchmark/benchmark.js]: Use `random/array/uniform`.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/examples/index.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var randu = require( '@stdlib/random/base/randu' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/examples/index.js]: Use `random/array/uniform`.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/lib/main.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+*
+* @example
+* var v = mean( NaN );
+* // returns NaN
+*/
+function mean( lambda ) {
+	if ( isnan( lambda ) || lambda < 0.0 ) {
+		return NaN;
+	}
+	return 1 / expm1(lambda);
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/lib/main.js]:
```suggestion
	return 1 / expm1( lambda );
```

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/lib/main.js [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var expm1 = require( '@stdlib/math/base/special/expm1' );
+
+
+// MAIN //
+
+/**
+* Returns the expected value of a Planck distribution.
+*
+* @param {number} lambda - shape parameter
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/lib/main.js]:
```suggestion
* @param {NonNegativeNumber} lambda - shape parameter
```

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/test/fixtures/python/REQUIRE [hunk 9]
```diff
@@ -0,0 +1,3 @@
+numpy 2.2.0
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/test/fixtures/python/REQUIRE]: What is this file? Python doesn't have a "REQUIRE" file.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/test/fixtures/python/runner.py [hunk 10]
```diff
@@ -0,0 +1,71 @@
+# !/usr/bin/env python
+#
+# @license Apache-2.0
+#
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/test/fixtures/python/runner.py:4]: Not clear why your runner diverges from our snippet: https://github.com/stdlib-js/stdlib/blob/develop/tools/snippets/test/fixtures/python/runner.py.

File: lib/node_modules/@stdlib/stats/base/dists/planck/mean/test/fixtures/python/runner.py [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+        Shape parameter.
+    name : str
+        Output filename.
+    Examples
+    --------
+    >>> lam = np.random.rand(1000) * 20
+    >>> gen(x, lam, "data.json")
+    """
+    # Compute mean values:
+    z = np.array([planck.mean(lami) for lami in lam])
```
Comment (kgryte) [inline lib/node_modules/@stdlib/stats/base/dists/planck/mean/test/fixtures/python/runner.py]: Why is this necessary? You can provide a NumPy array directly to `planck.mean`.

**Review by kgryte — CHANGES_REQUESTED** [review]
Left another round of review comments.

**Review by kgryte — APPROVED** [review]
This now LGTM. Thanks for working on this @Jaysukh-409.
